README.md

```diff
@@ -65,14 +65,15 @@ A curated list of anime streaming/downloading websites.
 * [station.f-n.me](https://station.f-n.me/)
 * [thehylia.com](https://anime.thehylia.com/)
 * [watchcartoononline.io](https://www.watchcartoononline.io/)
+* [turkanime.net](https://turkanime.net/) - Contains Turkish Subs
+* [tranimeizle.net](https://tranimeizle.net) - Contains Turkish Subs
 
 ### Anime (mobile app)
 * [#Animania](http://www.mobi24.net/#anime)
 
 ### Torrents
 * [animetosho.org](https://animetosho.org) - Torrent, NZB, DDL
 * [anidex.info](https://anidex.info/)
-* [animetorrents.me](https://animetorrents.me/)
```

Comment:
why remove?

Reply:
I did because you already created a section for private trackers right?

Reply:
ah okay yeah this seems good